solr/modules/ltr/src/java/org/apache/solr/ltr/interleaving/LTRInterleavingScoringQuery.java

```diff
@@ -25,7 +26,7 @@
 public class LTRInterleavingScoringQuery extends LTRScoringQuery {
 
   // Model was picked for this Docs
-  private Set<Integer> pickedInterleavingDocIds;
+  private Set<Integer> pickedInterleavingDocIds = Collections.emptySet();
```

Comment:
This seems to be another way to avoid the NPE (thanks for adding the tests!) and it would remove the need for `*TransformerFactory` changes ...

Though maybe it's a bit unconventional/controversial to initialise like this? What do you think?